src/components/page/App/AppCarousel.svelte

```diff
@@ -0,0 +1,227 @@
+<script>
+  import Arrow from "../../images/Arrow.svelte";
+  import { swipe } from "svelte-gestures";
+
+  const cards = [
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card1.png",
+      text: "Stay on top of your balance",
+    },
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card2.png",
+      text: "All your transactions in one place",
+    },
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card3.png",
+      text: "See the impact your super is having",
+    },
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card4.png",
+      text: "Easily share your details with employers",
+    },
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card5.png",
+      text: "Quick and easy login",
+    },
+    {
+      image:
+        "https://res.cloudinary.com/future-super/image/upload/v1690520834/app-card6.png",
+      text: "Easy access to important info",
+    },
+  ];
+
+  let innerWidth;
+  let scrollPosition = 0;
+  let cardWidth = innerWidth > 900 ? 500 : 400;
+
+  const handleMoveLeft = () => {
+    scrollPosition = Math.min(scrollPosition + cardWidth, 0);
+  };
+
+  const handleMoveRight = () => {
+    scrollPosition = Math.max(
+      scrollPosition - cardWidth,
+      -cardWidth * (cards.length - 1)
+    );
+  };
+
+  const swipeHandler = (event) => {
+    if (event.detail.direction === "right") {
+      // directions swapped for intuitive behaviour
+      handleMoveLeft();
+    } else {
+      handleMoveRight();
+    }
+  };
+
+  const resetCarousel = () => {
+    scrollPosition = 0;
+  };
+
+  $: if (innerWidth > 900) {
+    cardWidth = 500;
+    resetCarousel();
+  } else {
+    cardWidth = 400;
+    resetCarousel();
+  }
```

Comment:
Changes `cardWidth` with window resizing. Feels a bit hacky, but it works 🤷‍♀️

Reply:
Nothing wrong with doing it this way 👍